protocol/app/upgrades.go

```diff
@@ -37,6 +39,20 @@ func (app *App) setupUpgradeHandlers() {
 			app.VaultKeeper,
 		),
 	)
+	if app.UpgradeKeeper.HasHandler(v6_0_0_testnet_fix.UpgradeName) {
+		panic(fmt.Sprintf("Cannot register duplicate upgrade handler '%s'", v6_0_0_testnet_fix.UpgradeName))
+	}
+	app.UpgradeKeeper.SetUpgradeHandler(
```

Comment:
Here and below could we add brief comment that this upgrade handler is no-op for production